src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -144,8 +159,12 @@ def load_arguments(self, _):
         c.argument('disable_local_auth', arg_type=get_three_state_flag(), help='Disable all authentication methods other than AAD authentication.')
         c.argument('retention_days', arg_type=retention_days_arg_type)
         c.argument('enable_purge_protection', options_list=['--enable-purge-protection', '-p'], arg_type=get_three_state_flag(), help='Property specifying whether protection against purge is enabled for this App Configuration store. Setting this property to true activates protection against purge for this App Configuration store and its contents. Enabling this functionality is irreversible.')
+        c.argument('replica_name', arg_type=store_creation_replica_name_arg_type, help='Name of the replica of the App Configuration store.')
```

Comment:
Correct me if I am wrong, but it seems we are specifying help text twice?

Reply:
You are right, updated